00_Base/src/model/OcpiReservation.ts

```diff
@@ -0,0 +1,69 @@
+import { IsNotEmpty, IsString, MaxLength } from 'class-validator';
+import {
+  BelongsTo,
+  Column,
+  DataType,
+  ForeignKey,
+  Model,
+  Reservation,
+  Table,
+} from '@citrineos/data';
+import { CountryCode } from '../util/util';
+import { Enum } from '../util/decorators/enum';
+import { OcpiLocation } from './OcpiLocation';
+
+export enum OcpiReservationProps {
+  partyId = 'partyId',
+  countryCode = 'countryCode',
+  reservationId = 'reservationId',
+  locationId = 'locationId',
+  location = 'location',
+  coreReservationId = 'coreReservationId',
+  coreReservation = 'coreReservation',
+}
+
+@Table
+export class OcpiReservation extends Model {
```

Comment:
Do we need to add EVSEID here as a column? It's an optional in the incoming object, and it seems right now we just throw it away.  As well as `authorization_reference`

Reply:
Thank you, Christian! I added them in the table. I was struggled with this since these 2 field are not used in other following reserve steps beside reserveNow, `authorization_reference` even not used in reserveNow. But it would be more extendable if we need them in the future.